Library/Homebrew/shims/super/cc

```diff
@@ -215,11 +211,6 @@ class Cmd
     when "-Xpreprocessor", "-Xclang"
       # used for -Xpreprocessor -fopenmp
       args << arg << enum.next
-    when /-mmacosx-version-min=(\d+)\.(\d+)/
-      if high_sierra_or_later? && Regexp.last_match(1) == "10" && Regexp.last_match(2).to_i < 9
-        arg = "-mmacosx-version-min=10.9"
-      end
-      args << arg
```

Comment:
We should probably keep this and change it to:

```ruby
    when /-mmacosx-version-min=(\d+)\.(\d+)/
      if Regexp.last_match(1) == "10" && Regexp.last_match(2).to_i < 14
        arg = "-mmacosx-version-min=10.14"
      end
      args << arg
```

(If we also want to drop Mojave then 10.14 above should be 10.15.)